pkg/ui/workspaces/cluster-ui/src/databases/util.ts

```diff
@@ -99,6 +99,10 @@ export function createNodesByRegionMap(
 ): Record<string, number[]> {
   const nodesByRegionMap: Record<string, number[]> = {};
   nodes.forEach((node: number) => {
+    // If the node's region doesn't exist skip it.
+    if (nodeRegions[node.toString()] === undefined) {
```

Comment:
Do we specifically want to check for `undefined` or do we want to expand to a nullish check?

Reply:
nullish check would be better

Reply:
Yup, done.